x-pack/legacy/plugins/ml/server/models/job_audit_messages/job_audit_messages.d.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { APICaller } from 'src/core/server';
+
+export function jobAuditMessagesProvider(
+  callAsCurrentUser: APICaller
+): {
+  getJobAuditMessages: (jobId: string | undefined, from: string | undefined) => any;
+  getAuditMessagesSummary: (jobIds: string[] | undefined) => any;
```

Comment:
Nit: Could make use of optional parameters like `(jobId?: string, from?: string)` and `(jobIds?: string[])`.

Reply:
646802be53d4ce19923a519f3591aff07b6cadb5